lib/compiler-finder.ts

```diff
@@ -129,14 +137,14 @@ export class CompilerFinder {
                                 });
                                 res.on('end', () => {
                                     try {
-                                        const compilers = JSON.parse(str).map(compiler => {
+                                        const compilers = (JSON.parse(str) as CompilerInfo[]).map(compiler => {
                                             // Fix up old upstream implementations of Compiler Explorer
                                             // e.g. https://www.godbolt.ms
                                             // (see https://github.com/compiler-explorer/compiler-explorer/issues/1768)
                                             if (!compiler.alias) compiler.alias = [];
                                             if (typeof compiler.alias == 'string') compiler.alias = [compiler.alias];
                                             // End fixup
-                                            compiler.exe = null;
+                                            compiler.exe = '/dev/null';
```

Comment:
Per chatting with @partouf. I think it's slightly better to use '/dev/null' here instead of an empty string or leaving it undefined/null, it should give us an easy to track down error if the compiler ever tries to exec it.